build/configs/imxrt1050-evk/nxp_demo/defconfig

```diff
@@ -296,7 +304,16 @@ CONFIG_IMXRT_AUTOMOUNT_USERFS=y
 CONFIG_IMXRT_AUTOMOUNT_USERFS_DEVNAME="/dev/smart0p8"
 CONFIG_IMXRT_AUTOMOUNT_USERFS_MOUNTPOINT="/mnt"
 # CONFIG_IMXRT_AUTOMOUNT_SSSRW is not set
-# CONFIG_IMXRT_AUTOMOUNT_ROMFS is not set
+CONFIG_IMXRT_AUTOMOUNT_ROMFS=y
```

Comment:
If ROMFS is required, you need to change the CONFIG_IMXRT_FLASH_PART_xxxxx configs to add a new partition for romfs.